pymess/utils/migrations.py

```diff
@@ -12,32 +11,68 @@
     "SyncEmailTemplates",
 ]
 
-
-def get_email_template_body_from_file(email_template_slug, locale):
+def get_email_template_body_from_file(email_template_slug, locale=None, variant=None):
     base_dir = settings.EMAIL_HTML_DATA_DIRECTORY
     if locale:
-        base_dir = os.path.join(settings.EMAIL_HTML_DATA_DIRECTORY, locale)
-    with codecs.open(os.path.join(
-            base_dir,
-            '{}.html'.format(email_template_slug)), 'r', encoding='utf-8-sig') as file:
+        base_dir = os.path.join(base_dir, locale)
+
+    if variant == 'B':
+        filename = f'{email_template_slug}_B.html'
+    else:
+        filename = f'{email_template_slug}.html'
```

Comment:
This is awful.

Don't expect variant to be anything. Expect variant either to be set or not, i.e. to be None. The suffix may be empty only if `variant == None`.